testvectors/json/test-boot-seed-invalid-missing.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Juts a minor comment on the file name: it could be 
test-boot-seed-missing-json

Or if we want an outer name wrap it could be
test-boot-seed-error-invalid-long.json
test-boot-seed-error-missing.json
..like that

Reply:
There is a convention that I followed for minting the file names.  If I change this one like you propose I'd have to change all the other.  I'd rather leave all as it is.

Reply:
Is it a quick one word change in the script or is it a huge change?

Reply:
all these files have been crafted by hand: no script is involved in their assembly.

Reply:
OK, understood. May be we just raise a github issue to tidy up when we do some further major additions to this code base (like adding Profile 3). This way tidy up is remembered but not necessarily needs to be done now..

Reply:
not sure that work is needed.  the convention may be weird (all conventions are, in their own way), but if it's applied consistently it's ok.